ui-tests/src/test/java/io/syndesis/qe/steps/other/ConditionalFlowSteps.java

```diff
@@ -112,7 +113,8 @@ public void configureConditionOnPosition(int index) {
     @When("^return to primary flow from integration flow$")
     public void goBackToPrimaryFlow() {
         openDropdownWithConditions();
-        new CommonSteps().clickOnLink("Back to Primary Flow");
+        $(EditFlowStepElements.FLOW_BACK_BUTTON).shouldBe(visible).click();
+        //new CommonSteps().clickOnLink("Back to Primary Flow");
```

Comment:
why is this commented? Can we remove it?

Reply:
Yep forgot to delete it after I confirmed the new way worked, thx for noticing